### ISSUE TITLE
New issue on `geneos init san` after #57 & #61

### ISSUE THREAD

**Issue by gvastel** [ITRS-Group/cordial#63]
Commands with USERNAME  & DIRECTORY provided are now marked as error, despite them following the documented syntax.
Also the 2nd example below used to  execute before #61 was applied.
```
$ geneos init san itrs /opt/itrs -g gw_test1:7102 -a ATTR_1=VAL_1 -e ENV_VAR_1=ENV_VAL_1 -t TYPE_1 -v VAR_1=abc -v string:VAR_2=xyz
2022-11-22T02:41:11Z FATAL: tools/geneos/cmd/init.go:273 cmd.initProcessArgs() > too many args: [itrs /opt/itrs]
```
and also
```
$ geneos init san itrs /opt/itrs -g gw_test1:7102 -a ATTR_1=VAL_1 -e ENV_VAR_1=ENV_VAL_1 -t TYPE_1 -v VAR_1=abc
2022-11-22T02:42:43Z FATAL: tools/geneos/cmd/init.go:273 cmd.initProcessArgs() > too many args: [itrs /opt/itrs]
```

**Comment by pgalbavy-itrs** [comment]
We can add (even more) explicit command line flags to take directory and username, so we don't rely on optional positional arguments and how they are handled in different circumstances. After v1.4.0 though

**Comment by gvastel** [comment]
Having flags rather than relying on the positional arguments sounds good.  that may also help deliver better / more targeted error messages.

**Comment by pgalbavy-itrs** [comment]
Support for different users has been removed in v1.5.0-dev for clarity. The options may have to be reworking. Later we can add the support back in a better way.
